docs/pages/services/onto-docker.md

```diff
@@ -34,27 +33,27 @@ git clone https://git.material-digital.de/apps/ontodocker.git
 cd ontodocker
 ```
 
-### 2. Copy compose template
+### 2. Connect to SSO Identity Provider (IDP)
 
-```bash
-# copy (or link) the compose file template
-cp docker-compose-prod.yml docker-compose.yml
-```
+If you alreday have a valid client_secrets.json already, you must copy it to ```./data/oidc/```
```

Comment:
```suggestion
> **Note:** If you have a valid client_secrets.json already, place it in `./data/oidc/`
{: .warning }
```

Reply:
The triple backticks are a mistake. Single backticks are the right way, I agree.